doc/en/capture.rst

```diff
@@ -115,6 +115,10 @@ same interface but allows to also capture output from
 libraries or subprocesses that directly write to operating
 system level output streams (FD1 and FD2).
 
+If the code under test writes non-textual data, you can capture this using
+the ``capfdbinary`` function argument which instead returns ``bytes`` from
```

Comment:
I would prefer to use "fixtures" instead of "function argument".

Reply:
👍 , while writing this I myself wrote "fixtures" but then matched it with the paragraph above -- I'll update both of them